tests/localosmosis/scripts/setup.sh

```diff
@@ -84,6 +92,28 @@ edit_config () {
     dasel put string -f $CONFIG_FOLDER/config.toml '.rpc.laddr' "tcp://0.0.0.0:26657"
 }
 
+create_two_asset_pool () {
+    # Create default pool
+    substring='code: 0'
+    string=$(osmosisd tx gamm create-pool --pool-file=nativeDenomPool.json --from $MONIKER --chain-id=$CHAIN_ID --home $OSMOSIS_HOME --keyring-backend=test -b block --yes  2>&1)
+    if [ "$string" != "${string%"$substring"*}" ]; then
+        echo "create two asset pool: successful"
+    else
+        create_two_asset_pool
```

Comment:
Does it make sense to have a retry limit of some kind to avoid it looping infinitely?

Reply:
I forgot to do this! Thanks for catching, just implemented